openedx/features/course_experience/static/course_experience/js/CourseHome.js

```diff
@@ -4,13 +4,12 @@ export class CourseHome {  // eslint-disable-line import/prefer-default-export
   constructor(options) {
     // Logging for course tool click events
     const $courseToolLink = $(options.courseToolLink);
-    $courseToolLink.on('click', () => {
-      const courseToolName = document.querySelector('.course-tool-link').text.trim().toLowerCase();
+    $courseToolLink.on('click', (event) => {
+      const courseToolName = event.srcElement.text.trim().toLowerCase();
```

Comment:
Actually, @andy-armstrong @HarryRein, do we need some namespacing on these tool id names to avoid conflicts with future tool plugins?  Maybe just 'edx.bookmarks' to keep it simple?  Or should we just ignore.